src/docs/develop/variables.md

```diff
@@ -44,17 +44,19 @@ Variables can span multiple lines. Press `Control+Enter` (`cmd+Enter` on Mac) in
+|--------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `RAILWAY_STATIC_URL`                 | The public domain, of the form `example.up.railway.app`                                                                                                                                              |
+| `RAILWAY_GIT_COMMIT_SHA`             | The git [SHA](https://docs.github.com/en/github/getting-started-with-github/github-glossary#commit) of the commit that triggered the deployment. Example: `d0beb8f5c55b36df7d674d55965a23b8d54ad69b` |
+| `RAILWAY_GIT_AUTHOR`                 | The user of the commit that triggered the deployment. Example: `gschier`                                                                                                                             |
+| `RAILWAY_GIT_BRANCH`                 | The branch that triggered the deployment. Example: `main`                                                                                                                                            |
+| `RAILWAY_GIT_REPO_NAME`              | The name of the repository that triggered the deployment. Example: `myproject`                                                                                                                       |
+| `RAILWAY_GIT_REPO_OWNER`             | The name of the repository owner that triggered the deployment. Example: `mycompany`                                                                                                                 |
+| `RAILWAY_GIT_COMMIT_MESSAGE`         | The message of the commit that triggered the deployment. Example: `Fixed a few bugs`                                                                                                                 |
+| `RAILWAY_HEALTHCHECK_TIMEOUT_SEC`    | The timeout length (in seconds) of healthchecks. Example: `300`                                                                                                                                      |
+| `RAILWAY_ENVIRONMENT`                | The railway environment for the deployment. Example: `production`                                                                                                                                    |
+| `RAILWAY_DEPLOYMENT_OVERLAP_SECONDS` | How long the old deploy will overlap with the newest one being deployed. Example: `0`                                                                                                                |
+| `RAILWAY_DOCKERFILE_PATH`            | The path to the Dockerfile to be used by the service, the default value is `Dockerfile`. Example: `Railway.dockerfile`                                                                               |
```

Comment:
Could you add a new section below this for "User-Provided Configuration Variables" and throw these in there?

This section ("Railway-Provided Variables") is for variables that Railway provides to services. `RAILWAY_DEPLOYMENT_OVERLAP_SECONDS` and `RAILWAY_DOCKERFILE_PATH` are settings that a user can provide to Railway.

Reply:
Hi @half0wl! Thank you for the suggestion 😄 

I think it could be useful to specify which is the default value of `RAILWAY_DEPLOYMENT_OVERLAP_SECONDS`, do you know what it's supposed to be?
Also, is there any other User-Provided Configuration variable I should add?

Reply:
> I think it could be useful to specify which is the default value of `RAILWAY_DEPLOYMENT_OVERLAP_SECONDS`, do you know what it's supposed to be?

Good call! The default value for `RAILWAY_DEPLOYMENT_OVERLAP_SECONDS` is `20`.

>  Also, is there any other User-Provided Configuration variable I should add?

These are the two major ones. There are some others that I'll confirm internally and create an issue for them. Happy to assign that to you if you'd like to contribute further!

Reply:
I just added the information about `RAILWAY_DEPLOYMENT_OVERLAP_SECONDS`' default value.

> Happy to assign that to you if you'd like to contribute further!

Sure, I'd love to help this project